server/data/hmppsAuthClient.ts

```diff
@@ -73,4 +93,20 @@ export default class HmppsAuthClient {
 
     return newToken.body.access_token
   }
+
+  async getCuriousClientToken(): Promise<string> {
```

Comment:
Method to get the Curious client token from HMPP Auth.
It works in exactly the same way as the `getSystemClientToken` method, in that it caches it in the token store, only requesting HMPPS Auth to issue it when we need it (ie. when it's not in the token store / already expired)